imports/client/components/AccountForm.tsx

```diff
@@ -246,6 +256,51 @@ const AccountForm = (props: AccountFormProps) => {
     return <div>loading...</div>;
   }
 
+  const SignInWithGoogleBlock = () => {
```

Comment:
It uses a couple of things from the form: the submit state (to disable widgets while the popup is open) and the error message (to show an error if sign-in fails).

Reply:
Ah sure. I think I'd still prefer to see this live at top-level - defining a component inside of another component strikes me as kind of weird. (I know that it'll have slightly weird behavior on render, since each render will re-create the function and require swapping the component out)